### DISCUSSION TITLE
Can we get a new Release?

### DISCUSSION THREAD

**Discussion by gerdemann** [WasmEdge/WasmEdge#3823] [General]
Hello,
I am currently stuck on a solved bug in master. Could we possibly get a new release? 😄

**Comment by hydai** [comment]
May I ask which commit you need to use?
We are working on some breaking changes, which will be not very soon.

**Comment by gerdemann** [comment]
It would be this commit:
https://github.com/WasmEdge/WasmEdge/commit/ed6c3fe5a38ce17c4b6bbf7c8fd462337d46844c

It would be this issue:
https://github.com/WasmEdge/WasmEdge/issues/3803

**Reply by hydai** [reply]
If you are talking about the stable diffusion plugin, then it's fine. The AI/LLM plugins, including whisper, llama.cpp, and stable diffusion, are updated once the verification is ready, which means their assets will be updated in the same release pages based on the `0.14.1_plugin_lts` branch. Currently, we are working on the verification stage.

**Reply by gerdemann** [reply]
Yes, I'm talking about the stable diffusion plugin.

**Reply by hydai** [reply]
The plugin assets have been updated; please refer to the 0.14.1 release page to download them.

**Comment by hydai** [comment]
We will continue on updating the release assets of the WASI-NN related plugins. Closing this.
